trim_lcnaf.rb

```diff
@@ -0,0 +1,23 @@
+File.open("updated_trimmed_lcnaf.yml", "a") do |trimmed_file|
+  File.open("lcnaf.skos.ndjson").each_line do |line|
+    line_hash = eval(line)
+
+    id = line_hash[:"@context"][:about]
+    term = ""
+
+    graph = line_hash[:"@graph"]
+    graph.each do |graph_entry|
+      if graph_entry[:"@type"] == "skos:Concept"
+        # set the skos:prefLabel
+        term = graph_entry[:"skos:prefLabel"]
+      elsif graph_entry[:"@type"] == "skosxl:Label"
+        term = graph_entry[:"skosxl:literalForm"]
+      end
+
+      if !term.empty?
+        trimmed_file << "\"#{term}\":\"#{id}\"\n"
+        break
```

Comment:
We just want the first term set?

Reply:
There was a conversation on the last iteration where @fenekku explained the structure of the NDJSON file. In short, each line had the term we actually wanted and then a change log of sorts and a bunch of other unnecessary stuff. I will link to it in a second.

Reply:
Okay, I'm having a hard time linking to the comment since it's out of date. There might be a work around, but I think the gist of it is what I was saying above.

Reply:
That's right. It's in this conversation: https://github.com/galterlibrary/digital-repository/pull/968#discussion_r770550775